R/util.R

```diff
@@ -346,8 +346,27 @@ list_to_text <- function(column, sep = ", "){
 
 #' concatinate vectors in a list
 #' @export
-list_concat <- function(list){
-  list(unlist(list))
+list_concat <- function(..., collapse = TRUE){
```

Comment:
With the analogy to paste(), I think it makes more sense to make default FALSE.
I would guess breaking backward compatibility on this function now should have only minimum impact yet.